src/tests/ftest/util/soak_utils.py

```diff
@@ -480,6 +477,55 @@ def launch_snapshot(self, pool, name):
         "<<<PASS %s: %s completed at %s>>>\n", self.loop, name, time.ctime())
 
 
+def launch_vmd_identify_check(self, name, results, args):
+    """Run dmg cmds to blink/check VMD leds.
+
+    Args:
+        self (obj): soak obj
+        name (str): name of dmg subcommand
+        results (queue): multiprocessing queue
+        args (queue): multiprocessing queue
+    """
+    status = True
+    self.dmg_command.json.value = True
+    failing_vmd = []
+    result = self.dmg_command.storage_query_list_devices()
+
+    data = json.loads(result.stdout_text)
+    resp = data['response']
+    uuids = []
+    for value in list(resp['host_storage_map'].values()):
+        if value['storage']['smd_info']['devices']:
+            for device in value['storage']['smd_info']['devices']:
+                uuids.append(device['uuid'])
+
+    self.log.info("VMD device UUIDs: %s", uuids)
```

Comment:
FYI, if you wanted to display more detail: https://github.com/daos-stack/daos/blob/master/src/tests/ftest/deployment/disk_failure.py#L44-L49